sonar-plugin-api/src/test/java/org/sonar/api/utils/text/JsonWriterTest.java

```diff
@@ -103,13 +105,13 @@ public void type_of_values() throws Exception {
   @Test
   public void ignore_null_values_by_default() throws Exception {
     writer.beginObject()
-      .prop("nullNumber", (Number) null)
-      .prop("nullString", (String) null)
-      .name("nullNumber").value((Number) null)
-      .name("nullString").value((String) null)
-      .name("nullDate").valueDate(null)
-      .name("nullDateTime").valueDate(null)
-      .endObject().close();
+        .prop("nullNumber", (Number) null)
```

Comment:
I do, but IDEA default formatting and eclipse do not work well together, so I need to do explicit formatting and I guess I forgot once